airflow/www/utils.py

```diff
@@ -252,12 +256,10 @@ def wrapper(*args, **kwargs):
             dag_id=request.args.get('dag_id'))
 
         if 'execution_date' in request.args:
-            log.execution_date = dateparser.parse(
-                request.args.get('execution_date'))
+            log.execution_date = timezone.parse(request.args.get('execution_date'))
 
-        with create_session() as session:
-            session.add(log)
-            session.commit()
+        session.add(log)
+        session.commit()
```

Comment:
Don't we need to close this session if we remove the `with` statement? Personally I like the `with` syntax a lot.

Reply:
Mmm this seems the rebase is not entirely working correctly here, I havent changed this code. Good catch.